index.html

```diff
@@ -167,49 +167,100 @@ <h3 class="recent-works__title">Multi-Post Stories Gain+Glory</h3>
+					<div class="about-me__languages">Languages</div>
+					<ul class="about-me__tags">
+						<li class="about-me__tag">JavaScript</li>
+						<li class="about-me__tag">Ruby</li>
+						<li class="about-me__tag">Html</li>
+						<li class="about-me__tag">css</li>
+					</ul>
 				</li>      
 				<li class="about-me__works">
-				  <img class="about-me__icon" src="images/frameworks.svg" alt="languages icon">
-				  <div class="about-me__languages">Frameworks</div>
-				  <ul class="about-me__tags">
-					<li class="about-me__tag">Bootstrap</li>
-					<li class="about-me__tag">Ruby on rails</li>
-					<li class="about-me__tag">RSpec</li>
-					<li class="about-me__tag">Capybara</li>
-					<li class="about-me__tag">Selenium</li>
-				  </ul>
+					<img class="about-me__icon" src="images/frameworks.svg" alt="languages icon">
+					<div class="about-me__languages">Frameworks</div>
+					<ul class="about-me__tags">
+						<li class="about-me__tag">Bootstrap</li>
+						<li class="about-me__tag">Ruby on rails</li>
+						<li class="about-me__tag">RSpec</li>
+						<li class="about-me__tag">Capybara</li>
+						<li class="about-me__tag">Selenium</li>
+					</ul>
 				</li>
 				<li class="about-me__works">
-				  <img class="about-me__icon" src="images/skills.svg" alt="languages icon">
-				  <div class="about-me__languages">Skills</div>
-				  <ul class="about-me__tags">
-					<li class="about-me__tag">Codekit</li>
-					<li class="about-me__tag">GitHub</li>
-					<li class="about-me__tag">Codepen</li>
-					<li class="about-me__tag">Gitlab</li>
-					<li class="about-me__tag">Terminal</li>
-				  </ul>
+					<img class="about-me__icon" src="images/skills.svg" alt="languages icon">
+					<div class="about-me__languages">Skills</div>
+					<ul class="about-me__tags">
+						<li class="about-me__tag">Codekit</li>
+						<li class="about-me__tag">GitHub</li>
+						<li class="about-me__tag">Codepen</li>
+						<li class="about-me__tag">Gitlab</li>
+						<li class="about-me__tag">Terminal</li>
+					</ul>
 				</li>
-			  </ul>
+			</ul>
+		</section>
+		<!-- Contact Form Section -->
+		<section id="contact-form">			
+			<form class="contact-form__form" action="https://formspree.io/f/mzbqrajl" method="POST">
+				<div class="contact-form__primary">
+					<h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+				</div>
+				<label for="name">
+					<input type="text" name="name" id="name" autocomplete="name" placeholder="Full Name" maxlength="30" class="contact-form__input" required>
+				</label>
+				<label for="email">
+					<input type="text" name="email" id="email" autocomplete="email" placeholder="Email address" class="contact-form__input" required>
+				</label>
+				<label for="textarea">
+					<textarea name="message" minlength="10" maxlength="500" required>Enter text here</textarea>
+				</label>
+                <center><button type="submit">Get in touch</button></center>			    
```

Comment:
- [ ] You did a fantastic job styling the form of your portfolio :+1:, but as the project requires the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) should be styled as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=IPAfT7AoceJlYmE2-0).

![Screenshot from 2023-02-21 23-12-00](https://user-images.githubusercontent.com/105117832/220519985-822c8524-062a-49d9-b9ed-f7afcbe9fbb2.png)